.github/ISSUE_TEMPLATE/---bug-report.md

```diff
@@ -11,3 +11,8 @@ REDAXO:
 PHP:
 Browser:
 AddOns:
+
+**Did it whoops? / Whoops erhalten?**
+
+Don't forget to insert the stacktrace to provide more details and recieve answers faster.
+Vergiss nicht, den Stracktrace hier einzufügen, um genauere Details zur Verfügung zu stellen und schneller eine Antwort zu erhalten.
```

Comment:
Ich würde den user auf den „copy“ button stubsen - sobald man diesen clicked hat man Fehlermeldung/stacktrace (und soweit ich mich erinnere auch sysreport) alles in der zwischenablage

Reply:
![image](https://user-images.githubusercontent.com/3855487/76145654-c2043e00-608b-11ea-9fd6-c137b0965af2.png)

![image](https://user-images.githubusercontent.com/3855487/76145658-c92b4c00-608b-11ea-82ce-d6aa69e030a4.png)



**ParseError:** syntax error, unexpected 'if' (T_IF)
**File:** redaxo/src/addons/yrewrite_url_redirect/boot.php
**Line:** 3

<details>
<summary>Stacktrace</summary>

| Function                 | File                                     | Line     |
| ------------------------ | ---------------------------------------- | -------- |
| rex_package->includeFile | redaxo/src/core/lib/packages/package.php | 346      |
| rex_package->boot        | redaxo/src/core/packages.php             | 26       |
| {closure}                | redaxo/src/core/lib/util/timer.php       | 61       |
| rex_timer::measure       | redaxo/src/core/packages.php             | 28       |
| include_once             | redaxo/src/core/backend.php              | 185      |
| require                  | redaxo/src/core/boot.php                 | 135      |
| require                  | redaxo/index.php                         | 9        |

</details>
<details>
<summary>System report (REDAXO 5.8.0, PHP 7.3.14)</summary>

| REDAXO        |            |
| ------------: | :--------- |
|       Version | 5.8.0      |


| PHP           |            |
| ------------: | :--------- |
|       Version | 7.3.14     |
|       OPcache | no         |
|        Xdebug | no         |


| Database      |                            |
| ------------: | :------------------------- |
|       Version | 5.7.29                     |
| Character set | latin1 (latin1_swedish_ci) |


| Server        |            |
| ------------: | :--------- |
|            OS | Linux      |
|          SAPI | cgi-fcgi   |
|     Webserver | Apache     |


| Request       |                      |
| ------------: | :------------------- |
|       Browser | Chrome/80.0.3987.132 |
|      Protocol | HTTP/1.1             |
|         HTTPS | yes                  |


| Packages                         |             |
| -------------------------------: | :---------- |
|                          adminer | 1.7.0       |
|                         api_docs | 1.1.0       |
|                           backup | 2.4.0       |
|                         be_style | 2.8.0       |
|              be_style/customizer | 2.8.0       |
|                  be_style/redaxo | 2.8.0       |
|                          bloecks | 2.0.1       |
|                 bloecks/cutncopy | 2.0.1       |
|                bloecks/dragndrop | 2.0.1       |
|                   bloecks/status | 2.0.1       |
|                         ckeditor | 4.11.3.1    |
|                   cookie_consent | 3.0.0-beta3 |
|                        developer | 3.8.0       |
|                          install | 2.5.0       |
|                    media_manager | 2.7.0       |
|                        mediapool | 2.6.0       |
|                         metainfo | 2.5.0       |
|                      multiupload | 2.1.3       |
| multiupload/upload_precompressor | 1.0         |
|               navigation_factory | 2.0.1       |
|                        phpmailer | 2.6.0       |
|                          project | dev         |
|                 quick_navigation | 3.7.1       |
|                        redactor2 | 4.0.2       |
|                        structure | 2.8.0       |
|                structure/content | 2.8.0       |
|                            theme | 1.1.1       |
|                              url | 1.0.1       |
|                            users | 2.5.2       |
|                            yform | 3.2         |
|                       yform/docs | 3.2         |
|                      yform/email | 3.2         |
|                    yform/manager | 3.2         |
|                      yform/tools | 3.2         |
|                  yform_usability | 1.4         |
|                         yrewrite | 2.6         |
|            yrewrite_url_redirect | 1.1         |

</details>